diff --git a/node_modules/@changesets/cli/dist/cli.cjs.dev.js b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
index 5d48c11..7caf2cd 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.dev.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
@@ -485,7 +485,7 @@ async function version(cwd, options, config) {
 // @ts-ignore
 var isCI = !!(isCI$1 || process.env.GITHUB_ACTIONS);
 
-const npmRequestLimit = pLimit(40);
+const npmRequestLimit = pLimit(5);
 
 function jsonParse(input) {
   try {
diff --git a/node_modules/@changesets/cli/dist/cli.cjs.prod.js b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
index 0fb53aa..cd2ab58 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.prod.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
@@ -310,7 +310,7 @@ async function version(cwd, options, config) {
 
 var isCI = !(!isCI$1 && !process.env.GITHUB_ACTIONS);
 
-const npmRequestLimit = pLimit(40);
+const npmRequestLimit = pLimit(5);
 
 function jsonParse(input) {
   try {
