diff --git a/node_modules/@changesets/cli/dist/cli.cjs.dev.js b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
index e539f1b..14c6c05 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.dev.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
@@ -587,7 +587,7 @@ const getLastJsonObjectFromString = str => {
   return null;
 };
 
-const npmRequestLimit = pLimit__default['default'](40);
+const npmRequestLimit = pLimit__default['default'](3);
 const npmPublishLimit = pLimit__default['default'](10);
 
 function jsonParse(input) {
diff --git a/node_modules/@changesets/cli/dist/cli.esm.js b/node_modules/@changesets/cli/dist/cli.esm.js
index d1cf1fc..c25d64f 100644
--- a/node_modules/@changesets/cli/dist/cli.esm.js
+++ b/node_modules/@changesets/cli/dist/cli.esm.js
@@ -565,7 +565,7 @@ const getLastJsonObjectFromString = str => {
   return null;
 };
 
-const npmRequestLimit = pLimit(40);
+const npmRequestLimit = pLimit(3);
 const npmPublishLimit = pLimit(10);
 
 function jsonParse(input) {
