diff --git a/node_modules/@changesets/cli/dist/cli.cjs.dev.js b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
index 9f43a72..b64b8e0 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.dev.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
@@ -550,7 +550,7 @@ async function version(cwd, options, config) {
 // @ts-ignore
 var isCI = !!(isCI__default['default'] || process.env.GITHUB_ACTIONS);
 
-const npmRequestLimit = pLimit__default['default'](40);
+const npmRequestLimit = pLimit__default['default'](3);
 
 function jsonParse(input) {
   try {
diff --git a/node_modules/@changesets/cli/dist/cli.cjs.prod.js b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
index 9731f22..2830245 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.prod.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
@@ -315,7 +315,7 @@ async function version(cwd, options, config) {
 
 var isCI = !(!isCI__default.default && !process.env.GITHUB_ACTIONS);
 
-const npmRequestLimit = pLimit__default.default(40);
+const npmRequestLimit = pLimit__default.default(3);
 
 function jsonParse(input) {
   try {
diff --git a/node_modules/@changesets/cli/dist/cli.esm.js b/node_modules/@changesets/cli/dist/cli.esm.js
index 674ab7f..20b77ff 100644
--- a/node_modules/@changesets/cli/dist/cli.esm.js
+++ b/node_modules/@changesets/cli/dist/cli.esm.js
@@ -527,7 +527,7 @@ async function version(cwd, options, config) {
 // @ts-ignore
 var isCI = !!(isCI$1 || process.env.GITHUB_ACTIONS);
 
-const npmRequestLimit = pLimit(40);
+const npmRequestLimit = pLimit(3);
 
 function jsonParse(input) {
   try {
