diff --git a/node_modules/@changesets/cli/dist/cli.cjs.dev.js b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
index f771824..a2fb0ce 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.dev.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.dev.js
@@ -576,7 +576,7 @@ async function version(cwd, options, config) {
 // @ts-ignore
 var isCI = !!(isCI__default['default'] || process.env.GITHUB_ACTIONS);
 
-const npmRequestLimit = pLimit__default['default'](40);
+const npmRequestLimit = pLimit__default['default'](3);
 const npmPublishLimit = pLimit__default['default'](10);
 
 function jsonParse(input) {
diff --git a/node_modules/@changesets/cli/dist/cli.esm.js b/node_modules/@changesets/cli/dist/cli.esm.js
index 40bcc31..4b376a3 100644
--- a/node_modules/@changesets/cli/dist/cli.esm.js
+++ b/node_modules/@changesets/cli/dist/cli.esm.js
@@ -553,7 +553,7 @@ async function version(cwd, options, config) {
 // @ts-ignore
 var isCI = !!(isCI$1 || process.env.GITHUB_ACTIONS);
 
-const npmRequestLimit = pLimit(40);
+const npmRequestLimit = pLimit(3);
 const npmPublishLimit = pLimit(10);
 
 function jsonParse(input) {
