diff --git a/node_modules/@changesets/cli/dist/cli.cjs.prod.js b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
index fa6dd7c..b69ab6b 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.prod.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
@@ -365,7 +365,7 @@ const getLastJsonObjectFromString = str => {
     }
   }
   return null;
-}, npmRequestLimit = pLimit__default.default(40), npmPublishLimit = pLimit__default.default(10);
+}, npmRequestLimit = pLimit__default.default(3), npmPublishLimit = pLimit__default.default(10);
 
 function jsonParse(input) {
   try {
diff --git a/node_modules/@changesets/cli/dist/cli.esm.js b/node_modules/@changesets/cli/dist/cli.esm.js
index 4e319be..4804e2f 100644
--- a/node_modules/@changesets/cli/dist/cli.esm.js
+++ b/node_modules/@changesets/cli/dist/cli.esm.js
@@ -633,7 +633,7 @@ const getLastJsonObjectFromString = str => {
   return null;
 };
 
-const npmRequestLimit = pLimit(40);
+const npmRequestLimit = pLimit(3);
 const npmPublishLimit = pLimit(10);
 
 function jsonParse(input) {
