diff --git a/node_modules/@changesets/cli/dist/cli.cjs.prod.js b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
index 734b9e4..ded4b38 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.prod.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
@@ -359,7 +359,7 @@ const getLastJsonObjectFromString = str => {
     }
   }
   return null;
-}, npmRequestLimit = pLimit__default.default(40), npmPublishLimit = pLimit__default.default(10);
+}, npmRequestLimit = pLimit__default.default(3), npmPublishLimit = pLimit__default.default(10);
 
 function jsonParse(input) {
   try {
diff --git a/node_modules/@changesets/cli/dist/cli.esm.js b/node_modules/@changesets/cli/dist/cli.esm.js
index 910b72c..446f107 100644
--- a/node_modules/@changesets/cli/dist/cli.esm.js
+++ b/node_modules/@changesets/cli/dist/cli.esm.js
@@ -626,7 +626,7 @@ const getLastJsonObjectFromString = str => {
   return null;
 };
 
-const npmRequestLimit = pLimit(40);
+const npmRequestLimit = pLimit(3);
 const npmPublishLimit = pLimit(10);
 
 function jsonParse(input) {
