diff --git a/node_modules/@changesets/cli/dist/cli.cjs.prod.js b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
index ee3d78e..b7e34a5 100644
--- a/node_modules/@changesets/cli/dist/cli.cjs.prod.js
+++ b/node_modules/@changesets/cli/dist/cli.cjs.prod.js
@@ -354,7 +354,7 @@ const getLastJsonObjectFromString = str => {
     }
   }
   return null;
-}, npmRequestLimit = pLimit__default.default(40), npmPublishLimit = pLimit__default.default(10);
+}, npmRequestLimit = pLimit__default.default(3), npmPublishLimit = pLimit__default.default(10);
 
 function jsonParse(input) {
   try {
diff --git a/node_modules/@changesets/cli/dist/cli.esm.js b/node_modules/@changesets/cli/dist/cli.esm.js
index 19806a8..72859ff 100644
--- a/node_modules/@changesets/cli/dist/cli.esm.js
+++ b/node_modules/@changesets/cli/dist/cli.esm.js
@@ -616,7 +616,7 @@ const getLastJsonObjectFromString = str => {
   return null;
 };
 
-const npmRequestLimit = pLimit(40);
+const npmRequestLimit = pLimit(3);
 const npmPublishLimit = pLimit(10);
 
 function jsonParse(input) {
