diff --git a/node_modules/@graphql-codegen/near-operation-file-preset/cjs/index.js b/node_modules/@graphql-codegen/near-operation-file-preset/cjs/index.js
index a0636cb..17ebece 100644
--- a/node_modules/@graphql-codegen/near-operation-file-preset/cjs/index.js
+++ b/node_modules/@graphql-codegen/near-operation-file-preset/cjs/index.js
@@ -19,7 +19,7 @@ exports.preset = {
         const fileName = options.presetConfig.fileName || '';
         const extension = options.presetConfig.extension || '.generated.ts';
         const folder = options.presetConfig.folder || '';
-        const importTypesNamespace = options.presetConfig.importTypesNamespace || 'Types';
+        const importTypesNamespace = options.presetConfig.importTypesNamespace ?? 'Types';
         const importAllFragmentsFrom = options.presetConfig.importAllFragmentsFrom || null;
         const { baseTypesPath } = options.presetConfig;
         if (!baseTypesPath) {
diff --git a/node_modules/@graphql-codegen/near-operation-file-preset/cjs/resolve-document-imports.js b/node_modules/@graphql-codegen/near-operation-file-preset/cjs/resolve-document-imports.js
index f8dae4c..50fde90 100644
--- a/node_modules/@graphql-codegen/near-operation-file-preset/cjs/resolve-document-imports.js
+++ b/node_modules/@graphql-codegen/near-operation-file-preset/cjs/resolve-document-imports.js
@@ -24,7 +24,7 @@ function resolveDocumentImports(presetOptions, schemaObject, importResolverOptio
                 ...documentFile.document,
                 definitions: [...documentFile.document.definitions, ...externalFragments.map(fragment => fragment.node)],
             };
-            if ((0, plugin_helpers_1.isUsingTypes)(externalFragmentsInjectedDocument, [], schemaObject)) {
+            if ((0, plugin_helpers_1.isUsingTypes)(externalFragmentsInjectedDocument, [], schemaObject) && schemaTypesSource.namespace) {
                 const schemaTypesImportStatement = (0, visitor_plugin_common_1.generateImportStatement)({
                     baseDir,
                     emitLegacyCommonJSImports: presetOptions.config.emitLegacyCommonJSImports,
diff --git a/node_modules/@graphql-codegen/near-operation-file-preset/esm/index.js b/node_modules/@graphql-codegen/near-operation-file-preset/esm/index.js
index 6c71580..4747c07 100644
--- a/node_modules/@graphql-codegen/near-operation-file-preset/esm/index.js
+++ b/node_modules/@graphql-codegen/near-operation-file-preset/esm/index.js
@@ -15,7 +15,7 @@ export const preset = {
         const fileName = options.presetConfig.fileName || '';
         const extension = options.presetConfig.extension || '.generated.ts';
         const folder = options.presetConfig.folder || '';
-        const importTypesNamespace = options.presetConfig.importTypesNamespace || 'Types';
+        const importTypesNamespace = options.presetConfig.importTypesNamespace ?? 'Types';
         const importAllFragmentsFrom = options.presetConfig.importAllFragmentsFrom || null;
         const { baseTypesPath } = options.presetConfig;
         if (!baseTypesPath) {
diff --git a/node_modules/@graphql-codegen/near-operation-file-preset/esm/resolve-document-imports.js b/node_modules/@graphql-codegen/near-operation-file-preset/esm/resolve-document-imports.js
index e8796cb..4332e0f 100644
--- a/node_modules/@graphql-codegen/near-operation-file-preset/esm/resolve-document-imports.js
+++ b/node_modules/@graphql-codegen/near-operation-file-preset/esm/resolve-document-imports.js
@@ -20,7 +20,7 @@ export function resolveDocumentImports(presetOptions, schemaObject, importResolv
                 ...documentFile.document,
                 definitions: [...documentFile.document.definitions, ...externalFragments.map(fragment => fragment.node)],
             };
-            if (isUsingTypes(externalFragmentsInjectedDocument, [], schemaObject)) {
+            if (isUsingTypes(externalFragmentsInjectedDocument, [], schemaObject) && schemaTypesSource.namespace) {
                 const schemaTypesImportStatement = generateImportStatement({
                     baseDir,
                     emitLegacyCommonJSImports: presetOptions.config.emitLegacyCommonJSImports,
