diff --git a/node_modules/babel-jest/build/index.js b/node_modules/babel-jest/build/index.js
index f74b673..283655b 100644
--- a/node_modules/babel-jest/build/index.js
+++ b/node_modules/babel-jest/build/index.js
@@ -208,7 +208,7 @@ function loadBabelOptions(
   jestTransformOptions
 ) {
   const {options} = loadBabelConfig(cwd, filename, transformOptions);
-  return addIstanbulInstrumentation(options, jestTransformOptions);
+  return addIstanbulInstrumentation(options, transformOptions);
 }
 
 async function loadBabelOptionsAsync(
@@ -218,7 +218,7 @@ async function loadBabelOptionsAsync(
   jestTransformOptions
 ) {
   const {options} = await loadBabelConfigAsync(cwd, filename, transformOptions);
-  return addIstanbulInstrumentation(options, jestTransformOptions);
+  return addIstanbulInstrumentation(options, transformOptions);
 }
 
 const createTransformer = userOptions => {
@@ -320,11 +320,14 @@ const createTransformer = userOptions => {
       );
     },
 
-    process(sourceText, sourcePath, transformOptions) {
+    process(sourceText, sourcePath, config, transformOptions) {
       const babelOptions = loadBabelOptions(
-        transformOptions.config.cwd,
+        config.cwd,
         sourcePath,
-        mergeBabelTransformOptions(sourcePath, transformOptions),
+        mergeBabelTransformOptions(sourcePath, {
+          ...transformOptions,
+          config
+        }),
         transformOptions
       );
       const transformResult = (0, _core().transformSync)(
@@ -381,4 +384,4 @@ const transformer = {
   createTransformer
 };
 var _default = transformer;
-exports.default = _default;
+module.exports = _default;