diff --git a/dist/index.cjs b/dist/index.cjs
index 97af302..8715bd9 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -406,7 +406,7 @@ var _DefaultRenderer = class {
     }
     str = `${icon} ${str}`;
     let parsedStr;
-    let columns = process.stdout.columns ? process.stdout.columns : 80;
+    let columns = process.stdout.columns ? process.stdout.columns : process.platform === 'win32' ? 1000 : 80;
     columns = columns - level * this.options.indentation - 2;
     switch (this.options.formatOutput) {
       case "truncate":
