diff --git a/node_modules/typescript-json-schema/dist/typescript-json-schema.js b/node_modules/typescript-json-schema/dist/typescript-json-schema.js
index d5aa7f7..b5f0978 100644
--- a/node_modules/typescript-json-schema/dist/typescript-json-schema.js
+++ b/node_modules/typescript-json-schema/dist/typescript-json-schema.js
@@ -302,6 +302,8 @@ var annotationKeywords = {
     description: true,
     default: true,
     examples: true,
+    // https://github.com/YousefED/typescript-json-schema/issues/431
+    exampleMarkdown: true,
     $ref: true,
 };
 var subDefinitions = {
