diff --git a/node_modules/nextra-theme-docs/dist/index.d.ts b/node_modules/nextra-theme-docs/dist/index.d.ts
index 6fffd4e..693c324 100644
--- a/node_modules/nextra-theme-docs/dist/index.d.ts
+++ b/node_modules/nextra-theme-docs/dist/index.d.ts
@@ -100,4 +100,5 @@ declare const createLayout: (opts: DocsLayoutProps, config: DocsThemeConfig) =>
     getLayout(page: any): JSX.Element;
 };
 
+export declare const components: Record<string, any>;
 export { DocsThemeConfig, Meta, PageTheme, createLayout as default };
diff --git a/node_modules/nextra-theme-docs/dist/index.js b/node_modules/nextra-theme-docs/dist/index.js
index d509dde..5d2a15d 100644
--- a/node_modules/nextra-theme-docs/dist/index.js
+++ b/node_modules/nextra-theme-docs/dist/index.js
@@ -2342,6 +2342,7 @@ var createLayout = (opts, config) => {
   return Page;
 };
 var src_default = createLayout;
+export const components = getComponents({ index: 0 })
 export {
   src_default as default
 };
